README.md

```diff
@@ -744,6 +744,12 @@ This will return the user to "https://www.example.org/logged_out.html"
 after the logout operation has completed.
 
 
+## Invalidating session
+It is possible to invalidate the current mod_auth_mellon session, without calling SLO. 
+The mod_auth_mellon cookie session will be invalidate and session will be removed from mod_auth_mellon cache.
```

Comment:
invalidate and session -> invalidated and the session
from mod_auth_mellon -> from the mod_auth_mellon